articles/appliance/geo-ha2.md

```diff
@@ -0,0 +1,48 @@
+# Auth0 Appliance: High Availability Geo Cluster (Geo HA)
+
+The high availability geo cluster is an Appliance implementation that provides data center redundancy and automatic failure handling. This is the highest form of Appliance availability offered by Auth0.
+
+## Overview
+
+Auth0 adds to the single data center high availability solution by extending the cluster with a geographically distributed data center with a **maximum latency between the two data centers of no more than 100ms (round trip)**. The result is the high availability geo cluster, which is an active hot standby configuration with automated failure handling that can survive a regional outage.
+
+## Standard Configuration
+
+![](/media/articles/appliance/geo-ha.png)
+
+The standard configuration is a stretch cluster that consists of the following pieces:
+
+* one geographically-aware global load balancer/DNS failover configuration;
+* one primary data center with three Appliance instances;
+* one standby data center with three Appliance instances;
+* one arbiter, a seventh instance that is located in its own data center.
+
+The standby data center instances possess the same Appliance configuration as the primary data center instances, and continuous synchronization ensure that the data on the primary and standby data centers mirror each other.
+
+The arbiter does not store data or execute application logic, but acts as a witness between the primary and standby data centers. By independently verifying if a data center is down or not, it prevents both from becoming active (such a scenario is known as the "split-brain" condition).
+
+> Ports 27017 and 7777 must be open between all instances in the cluster.
+
+### Application Tier
+
+Auth0 requires the use of a geographically-aware load balancer or DNS failover solution that prefers to serve application requests using the primary data center, despite the fact that the Appliance instances in the hot standby data center are active and able to serve the requests.
+
+The application tier remains unaware of the locality of the primary data node, whereas the data layer resolves the location of the primary node (the only node that receives application queries). Additionally, the current implementation does not perform sharing or reading from a secondary replica, so all read and write activities are served by the primary node. For example, if the data centers for the geo cluster are in California (CA) and Texas (TX) and the ones in TX are active, any request serviced by the CA nodes at the application level requires data to then be written to a node in TX. This generally results in poorer performance due to the requests (and resulting round-trips) required to obtain the necessary data. Using a geographically-aware load balancer or DNS failover solution mitigates this performance issue.
```

Comment:
Additionally, the current implementation does not perform sharing or reading from a secondary replica, so all read and write activities are served by the primary node. => I think this gives away a little too much. can we just say `All read and write activities are served by the primary node`.

Also let's use Washington and Colorado in the example because TX and CA relate specifically to the AAA configuration, I'd like to make more ambiguous.